lib/ansible/modules/web_infrastructure/acme_certificate.py

```diff
@@ -355,14 +366,25 @@ def __init__(self, module):
         self.finalize_uri = self.data.get('finalize_uri') if self.data else None
 
         # Make sure account exists
-        contact = []
-        if module.params['account_email']:
-            contact.append('mailto:' + module.params['account_email'])
-        self.changed = self.account.init_account(
-            contact,
-            agreement=module.params.get('agreement'),
-            terms_agreed=module.params.get('terms_agreed')
-        )
+        modify_account = module.params['modify_account']
+        if modify_account or self.version > 1:
+            contact = []
+            if module.params['account_email']:
+                contact.append('mailto:' + module.params['account_email'])
+            self.changed = self.account.init_account(
+                contact,
+                agreement=module.params.get('agreement'),
+                terms_agreed=module.params.get('terms_agreed'),
+                allow_creation=modify_account,
+                update_contact=modify_account
+            )
+        else:
+            # This happens iff modify_account is False and the ACME v1
```

Comment:
typo, and I see a discrepancy in the sentence: "This happens if modify account is False *or* the ACME v1 is used

Reply:
I've changed it to "if" since "if and only if" is too long ;) I've also rebased to get rid of the conflicts (?).